styles/styles.css

```diff
@@ -354,6 +351,14 @@ main .section.full-width-banner.top-edge {
   padding-bottom:  var(--gutter-space-triangle-mobile);
 }
 
+footer {
+  background-color: var(--teams-background-color);
```

Comment:
Plz take a pull from main branch, the variable name (started with --teams) has been changed now.

Reply:
done